shell/common/shell.cc

```diff
@@ -1229,4 +1237,15 @@ Rasterizer::Screenshot Shell::Screenshot(
   return screenshot;
 }
 
+bool Shell::WaitForFrameRender(fml::TimeDelta timeout) {
+  std::unique_lock<std::mutex> lock(waiting_for_frame_mutex_);
+  task_runners_.GetUITaskRunner()->AwaitTask(
+      [this] { engine_->GetAnimator()->ForceVSync(); });
```

Comment:
What happens if we don't force vsync?

Reply:
It actually works because at the point that we are syncing the vsync is setup.
1) This is safer since we are forcing a render.
2) I'm hoping to reduce latency.  in a worse case scenario this could save us 1/60 seconds.